<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home | Pacman</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgresources.qrc">
    <normaloff>:/img/pacman/1.png</normaloff>:/img/pacman/1.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>448</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/background);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/pacman_logo);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="initial_gif_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="developer_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Developed by angelortizv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playgame_button">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>300</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PLAY GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="highscores_button">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>360</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>HIGH SCORES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>410</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../imgresources.qrc"/>
 </resources>
 <connections/>
</ui>
